horovod/tensorflow/__init__.py

```diff
@@ -282,14 +286,17 @@ def DistributedOptimizer(optimizer, name=None, use_locking=False, device_dense='
         Treat all sparse gradients as dense tensors.  This can help improve
         performance and memory utilization if the original sparse gradient
         has high density.  Defaults to false.
+      sum_over_ranks: If True, computes the sum of the gradients
```

Comment:
Please add new line before `If True`, to align with previous args.

Reply:
Fixed in ffda60499308d659f7502b762e3e2371a4c23750